sequencing/src/main/java/org/biojava/bio/program/abi/ABITrace.java

```diff
@@ -1,3 +1,5 @@
+.
+ .
```

Comment:
Not sure where these two new lines came from, they are failing the Travis build.